website/docs/r/compute_instance_group.html.markdown

```diff
@@ -97,3 +97,11 @@ exported:
 * `self_link` - The URI of the created resource.
 
 * `size` - The number of instances in the group.
+
+## Import
+
+Instance group can be imported using the `zone` and `name`, e.g.
```

Comment:
Just wondering if you and @danawillow have talked about what we want to do for `region` and `zone` across all importable resources - we've checked against the provider region, or against all the provider region's zones in the past to find objects by Id alone.

I like this approach a lot - it means we don't clutter read with parsing logic by using an import func, we don't need to write migration functions, and it lets users import resources from more than 1 region/1 set of zones inside a single provider. Are we going to use the same approach across all importable resources going forward and do we have a plan to bring it to older ones?